lightly/models/modules/nn_memory_bank.py

```diff
@@ -38,9 +39,16 @@ class NNMemoryBankModule(MemoryBankModule):
     """
 
     def __init__(self, size: int = 2**16):
+        if size <= 0:
+            raise ValueError(f"Memory bank size must be positive, got {size}.")
         super(NNMemoryBankModule, self).__init__(size)
 
-    def forward(self, output: torch.Tensor, update: bool = False):
+    def forward(
+        self,
+        output: torch.Tensor,
+        labels: Optional[torch.Tensor] = None,
```

Comment:
Why was the labels argument added?

Reply:
I have no idea. Will remove it again.

Reply:
It's because of mismatching signatures. I added a TODO instead.